### PR TITLE
Update UG & DG

### LINKED ISSUE #35: Update UG and DG

### PR DESCRIPTION
**Pull request by CSmortal**
Closes #35
Changed/added use cases in DG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -61,25 +74,128 @@ Taskify is a **desktop app for students manage their tasks (academics/personal/C
...
+Format: `add n/NAME desc/DESCRIPTION a/ADDRESS date/DATE [t/TAG]…`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `example 1`
-* `example 2`
+* `add n/myTask1 desc/1st Task a/John street, block 123, #01-01 date/2020-04-13`
```
Comment (noelmathewisaac) [inline docs/UserGuide.md:105]: can you replace lines 1-2 with these
```
---
layout: page
title: User Guide
```
I think this is needed for the website to display UG and DG links and you removed these lines in the previous PR.
Currently, if you check the netlify build for the website, there are no links to the UG and DG

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -311,14 +311,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **Extensions**
 
 * 2a. The list is empty.
-
-  Use case ends.
+    * 2a1. Taskify warns that the list is empty.
```
Comment (noelmathewisaac) [inline docs/DeveloperGuide.md:316]: Could you update the user stories to show the edit status and sort by deadline stories. Also, I think all the priorities should be High or Medium for the current user stories

**Review by noelmathewisaac — CHANGES_REQUESTED** [review]
Update User stories and UG first 2 lines

**Review by khiaxeng — APPROVED** [review]
LGTM

**Review by noelmathewisaac — CHANGES_REQUESTED** [review]
The website still does not show the link to DG. Can you change the DG from 
```
---
layout: page title: Developer Guide
---
``` 
to 
```
---
layout: page 
title: Developer Guide
---
```

**Review by noelmathewisaac — APPROVED** [review]
LGTM

**Pull request merged**